bricks/template/hooks/pre_gen.dart

```diff
@@ -31,3 +47,16 @@ Future<Map<String, dynamic>> addPermissionHandlerVariables() async {
   });
   return vars;
 }
+
+Future<void> deleteFileSystemEntities(List<FileSystemEntity> entities) async {
+  entities.forEach((entity) {
+    if (entity.existsSync()) {
+      print('Delete ' + entity.path);
+      if (entity is File) {
+        entity.deleteSync();
+      } else if (entity is Directory) {
+        entity.deleteSync(recursive: true);
+      }
+    }
+  });
+}
```

Comment:
I recommend creating a `hooks_util.dart` file to store all utility functions related to `hooks`. This will help keep the code organized and make it easier to maintain.

Reply:
@manh-t Updated in https://github.com/nimblehq/flutter-templates/pull/271/commits/919a1170fdcb8be37ce0e5eebaf07b3a3afbfa73